src/components/UserAccess/__tests__/user-access-actions.spec.ts

```diff
@@ -1,100 +1,111 @@
-import '@testing-library/jest-dom';
-import { renderHook } from '@testing-library/react-hooks';
+import { mockRoleBinding, mockRoleBindingWithoutUser } from '../../../__data__/rolebinding-data';
 import { useAccessReviewForModel } from '../../../utils/rbac';
-import { createUseWorkspaceInfoMock } from '../../../utils/test-utils';
-import { useSBRActions } from '../user-access-actions';
+import { useModalLauncher } from '../../modal/ModalProvider';
+import { useWorkspaceInfo } from '../../Workspace/useWorkspaceInfo';
+import { useRBActions } from '../user-access-actions';
 
+// Mock the dependencies
 jest.mock('../../../utils/rbac', () => ({
   useAccessReviewForModel: jest.fn(),
 }));
 
-const accessReviewMock = useAccessReviewForModel as jest.Mock;
+jest.mock('../../Workspace/useWorkspaceInfo', () => ({
+  useWorkspaceInfo: jest.fn(),
+}));
```

Comment:
same as above, you shouldn't need to mock useWorkspaceInfo hook

Reply:
Fixed